traffic_ops/traffic_ops_golang/parameter/parameters.go

```diff
@@ -21,6 +21,7 @@ package parameter
 
 import (
 	"errors"
+	"github.com/apache/trafficcontrol/lib/go-atscfg"
```

Comment:
These imports ought to be separate from the standard library

Reply:
Separated that import from the standard library in 63eb27d310